.github/workflows/lint.yaml

```diff
@@ -0,0 +1,155 @@
+name: lint
+on: [push, pull_request]
+jobs:
+  check-licences:
```

Comment:
This should ideally use/depend on the build step.

Reply:
Will do.

Reply:
Fixed!